### PR TITLE
Update FS-1012-caller-info-attributes.md

### PR DESCRIPTION
**Pull request by AviAvni**

### CODE REVIEW

File: RFCs/FS-1012-caller-info-attributes.md [hunk 1]
```diff
@@ -68,12 +68,50 @@ Can computation expression methods accepting caller info attributes?
 
 Yes: Computation Expressions accept caller info attributes no special implementation and there are tests in the PR.
 
+Check within an async expression ``async { ... }`` (whose desugaring has some implied lambda expressions)
+
+Yes: Ayync computation expression values works as expected no special implementation and there are tests in the PR.
```
Comment (dsyme) [inline RFCs/FS-1012-caller-info-attributes.md]: typo Ayync

File: RFCs/FS-1012-caller-info-attributes.md [hunk 2]
```diff
@@ -68,12 +68,50 @@ Can computation expression methods accepting caller info attributes?
...
+
 ## Feature Interaction - Quotations
 
 Check this feature works as expected with quotation literals
 
 Yes: Quotations works as expected no special implementation and there are tests in the PR.
 
+## Feature Interaction - Anonymous function
+
+Checl an anonymous lambda experession, e.g. ``(fun () -> ...)``.
```
Comment (dsyme) [inline RFCs/FS-1012-caller-info-attributes.md]: Typo Checl

File: RFCs/FS-1012-caller-info-attributes.md [hunk 3]
```diff
@@ -93,9 +131,3 @@ Some alternatives are:
 [unresolved]: #unresolved-questions
 
 [Query](https://github.com/Microsoft/visualfsharp/pull/1114/files#r66590569): Could you please also specify and test what happens when the call to GetCallerMemberName is made from
```
Comment (dsyme) [inline RFCs/FS-1012-caller-info-attributes.md]: You can delete this line too?

Comment (AviAvni) [inline RFCs/FS-1012-caller-info-attributes.md]: @dsyme Done

**Comment by AviAvni** [comment]
@dsyme It's OK now?

**Pull request merged**